packages/services/video-tools/video_utils/fal_video_analyzer.py

```
@@ -0,0 +1,574 @@
+        Args:
+            api_key: FAL API key (defaults to FAL_KEY env var)
+            model: Model to use for analysis (default: google/gemini-3-flash)
+
+        Raises:
+            ImportError: If fal-client is not installed
+            ValueError: If FAL_KEY is not set
+        """
+        if not FAL_AVAILABLE:
+            raise ImportError(
+                "FAL client not installed. Run: pip install fal-client"
+            )
```

Ruff (0.14.11)

Avoid specifying long messages outside the exception class

(TRY003)